src/clientsettingsdlg.h

```diff
@@ -97,6 +97,14 @@ public slots:
     void OnEnableOPUS64StateChanged ( int value );
     void OnCentralServerAddressEditingFinished();
     void OnNewClientLevelEditingFinished() { pSettings->iNewClientFaderLevel = edtNewClientLevel->text().toInt(); }
+    void OnInputGainTextChanged() {
```

Comment:
This is a bit long for the `.h` file...

Reply:
Moved it to the `.cpp` file and fixed a style issue as well.